### DISCUSSION TITLE
Error log screen

### DISCUSSION THREAD

**Discussion by ubuster** [cybertec-postgresql/pg_timetable#300] [General]
After program start without both parameters "log-file" and  "log-file-format" the screen log is formatted colorless, e.g.
2021-07-28 15:55:22.617←[32m [INFO] ←[0mDatabase connection established
2021-07-28 15:55:22.668←[32m [INFO] ←[0mAccepting asynchronous chains execution requests...
2021-07-28 15:55:22.669←[32m [INFO] [count:1] ←[0mRetrieve scheduled chains to run @reboot
2021-07-28 15:55:22.691←[32m [INFO] [count:1] ←[0mRetrieve interval chains to run
2021-07-28 15:55:22.739←[32m [INFO] [chain:1] ←[0mStarting chain

When both parameters used the screen log is colorfull, e.g green.

**Comment by pashagolub** [comment]
Hello,

would you please provide us with a command line showing this issue? Some screenshots may help as well.
What operating system you were using?

Thanks in advance!

**Reply by ubuster** [reply]
Hello,
I am a Windows user. After opening CMD screen I provide the right (colorfull) command 
_"pg_timetable postgresql://localhost:5501/postgres /log-file:d:/timetable4/tt_log.log /log-file-format:text -c klient"_ 
![pict2](https://user-images.githubusercontent.com/14990356/127615088-98f7cb3f-761e-4574-9dd0-0a970c515d17.jpg). 
Then I provide the easier (B/W) command _"pg_timetable postgresql://localhost:5501/postgres -c klient"_ 
![pict1](https://user-images.githubusercontent.com/14990356/127615149-4d0d90bd-0cf4-42f7-890f-3f26d5a65c27.jpg).

Hopefully this helps you to understand what I mean.
Thanks!

**Reply by pashagolub** [reply]
Thanks for your help.

I still can see color codes present but I'm not sure why the output isn't colored. Would you please try [PowerShell](https://github.com/PowerShell/PowerShell) or [Windows Terminal](https://github.com/microsoft/terminal) instead of `cmd` console?

My guess is `Ctrl + C` somehow breaks console output, and if you will try to launch the second command in the new `cmd` it will work as expected. Can you confirm that, please?

**Reply by ubuster** [reply]
Hello,
sorry for delaying with my answer. The output in [PowerShell](https://github.com/PowerShell/PowerShell) is good colored indeed.

**Comment by pashagolub** [comment]
I would consider this as a glitch of `cmd.exe` then... 🤷
